ivc/src/poseidon_params_8_56_5_3.rs

```diff
@@ -1008,9 +1008,9 @@ mod tests {
             Fp::from_str("1").unwrap(),
         ];
         let exp_output_str = [
-            "1f1a9ee0bafa6811f1f9bd1e9a7a1ffe6bed28608a11a07833058622f2089b0e",
```

Comment:
Do we have any other regression tests that make sure that other parameterizations were not changed in the previous PR that changed the poseidon spec in `poseidon/` crate?